src/parser/parser.ts

```diff
@@ -117,7 +117,7 @@ export const parsePage = async (
 	let pronunciation = "";
 	do {
 		// const targetSelector = `span[data-language-for-alternatives=${to}]`;
-		const targetSelector = `span[lang=${to}]`;
+		const targetSelector = `.ryNqvb`;
```

Comment:
This class name is automatically randomly generated when bundling the website. Every time Google Translate publishes a new version, it will be changed. We need a better selector that is more likely to be stable.

Reply:
In my newest commit I changed the targetSelector to `span[lang=${to}] > span > span` which should hopefully be more stable.